.storybook/misc.jsx

```diff
@@ -75,3 +75,29 @@ export const QuarterPickerWrapper = (props) => {
       </div>
   );
 };
+
```

Comment:
This is fine for now, but I would think about breaking `misc.jsx` out into smaller files before it becomes massive as we keep adding components.